src/java.base/share/classes/jdk/internal/foreign/Utils.java

```diff
@@ -273,4 +273,9 @@ public static <L extends MemoryLayout> L wrapOverflow(Supplier<L> layoutSupplier
     public static boolean containsNullChars(String s) {
         return s.indexOf('\u0000') >= 0;
     }
+
+    public static String toHexString(long value) {
```

Comment:
We intend to use this utility method in coming PRs.

Reply:
The String concat code is really good at optimizing, but this will require a new string be created.
$0.02,Roger

Reply:
Why not `HexFormat.of().withPrefix("0x").formatHexString(long)` I thought, then realized there's no method such method that takes a `long` so that the "0x" + hex digits can be done in a single step without concatenation. Seems there might be room for improvement here.